packages/js/src/utils/errors.ts

```diff
@@ -17,3 +25,18 @@ export class LogtoError extends Error {
     this.code = code;
   }
 }
+
+export type CallbackUriErrorCode = NormalizeKeyPaths<typeof callbackUriErrorCodes>;
+
+export class CallbackUriVerificationError extends Error {
```

Comment:
we can define a unified `LogtoError` and use `.` as the separator to leverage `NormalizeKeyPaths<>`.

Reply:
Updated, PTAL.